internal/kafka/internal/handlers/cluster.go

```diff
@@ -164,11 +169,19 @@ func (h clusterHandler) List(w http.ResponseWriter, r *http.Request) {
 				Page:  1,
 				Size:  int32(len(clusters)),
 				Total: int32(len(clusters)),
-				Items: []public.EnterpriseClusterListItem{},
+				Items: []public.EnterpriseCluster{},
 			}
 
 			for _, cluster := range clusters {
-				converted := presenters.PresentEnterpriseClusterListItem(*cluster)
+				consumedCapacity, consumedErr := h.getEnterpriseClusterConsumedStreamingUnitCountFor(cluster.ClusterID)
```

Comment:
This will result in `1+N` DB calls - we might need to reconsider this in the future when we notice perf issue

Reply:
:+1: . Could you open a jira issue to keep track of that?

Thanks

Reply:
Thanks :+1: